lib/cell/option.rb

```diff
@@ -0,0 +1,35 @@
+require "trailblazer/option"
+require "uber/callable"
+
+# DISCUSS: Should we move this to trb-option instead ?
+# This is identical to `Representable::Option`.
+module Cell
+  # Extend `Trailblazer::Option` to support static values as callables too.
+  class Option < ::Trailblazer::Option
```

Comment:
@apotonick This is identical to [Representable::Option](https://github.com/trailblazer/representable/blob/master/lib/representable/option.rb). 

Should we move `Cell::Option` and `Cell::Options` in trb-option instead ?

Reply:
Hm, do we have use on a generic level?

Reply:
Hm, kind of.

1. Handling static values might be needed somewhere else.
2. But `Cell::Options` isn't generic. We can keep it here.

Reply:
The question is, do we have the static requirement anywhere else (e.g. Reform)? If yes, we can extract it. On the other hand, we can always do that later. :yawning_face:

Reply:
Yep, we've the static requirement in reform, disposable and representer. Currently, it is done in representer itself and it's being used in other 2.

Reply:
On the other hand, supporting `Callable` objects between these 2 types of gems (representable, reform, cells etc and trb-activity, DSL etc) is little different i.e.
1. Extending `Uber::Callable` for identifying callable object.
2. trb-activity and friends doesn't need static type support.

So other way is we should not mix these 2 types in trb-option but keep separate. If it's ok, then we can keep `Cell::Option` here.

Reply:
1. Reg. `Uber::Callable`, that one we should dump ASAP. Isn't it better if we always treat an object `call`able, anyway?
2. Hm, but it also doesn't hurt to have it there instead of maintaining 4 different `Option`s, right?

Reply:
Hm yeah. The problem with treating every object callable is we can't distinguish between them and static options (some static options also includes modules, classes etc), apart from `call` method.

Reply:
Ahhh, ok right, so that was why I initially introduced `Uber::Callable`, damn! Hm... how could we do that in the future?

Ok, in that case, we won't move that behavior to TRB, for now. :cake:

Reply:
Correct (i.e. distinction between static and callable objects isn't needed in TRB). So we can merge this PR.

`Uber::Callable` is a nice alternative for `cells` and `representable friends` for now.
In future versions, maybe we can have restrictions on static options (or ways to avoid them) ?